client/src/app/components/ProjectTile/ProjectTile.js

```diff
@@ -8,13 +8,24 @@ const formatDescription = description =>
   description.length > 80 ? description.substring(0, 100) + "..." : description;
 
 const ProjectTile = props => {
-  const { name, description, level, stack, image = "#", requiredTeamSize } = props.project;
+  const { name, description, level, stack, requiredTeamSize } = props.project;
+  const generateRandomImage = () => {
```

Comment:
This would have been better to be placed completely outside the react component, as it doesn't use any data from it.